egg/platform_ppc.py

```diff
@@ -41,13 +43,33 @@ def ppc_vec_type(typ):
         return '__vector signed short'
     elif typ == 'u32':
         return '__vector unsigned int'
+    # elif typ == 'u64':
+    #     return '__vector unsigned long long'
```

Comment:
According to GCC documentation you can use for VSX 64-bits types, maybe something like this?

```python
def ppc_vec_type(typ, simd_ext):
    if typ == 'u8':
        return '__vector unsigned char'
    ...
    if typ == 'f64':
        if simd_ext == 'vmx':
            return '__vector double'
    ...
```